src/react/components/Link.test.tsx

```diff
@@ -33,7 +33,7 @@ describe("Link", () => {
     renderInHistory(history, <Component />);
     expect(screen.queryByText("I AM BAR")).toBeNull();
     expect(history.index).toBe(0);
-    screen.queryByTestId("link")?.click();
+    fireEvent.click(screen.queryByTestId("link")!);
```

Comment:
I just abbused non-null assertion. Let me know if you prefer decent null check condition for that.